### PR TITLE
Parameterize addons

### PR DESCRIPTION
**Pull request by anders-swanson**
Parameterize some addon functionality for CNI, CSI, and CCM.

### CODE REVIEW

File: ociocne/variables/variables.go [hunk 1]
```diff
@@ -145,11 +152,16 @@ func NewFromOptions(ctx context.Context, driverOptions *types.DriverOptions) (*V
...
 		CalicoTag:            options.GetValueFromDriverOptions(driverOptions, types.StringType, driverconst.CalicoTag, "calicoImageTag").(string),
 		CCMImage:             options.GetValueFromDriverOptions(driverOptions, types.StringType, driverconst.CCMImage, "ccmImage").(string),
-		ETCDImageTag:         options.GetValueFromDriverOptions(driverOptions, types.StringType, driverconst.ETCDImageTag, "etcdImageTag").(string),
-		CoreDNSImageTag:      options.GetValueFromDriverOptions(driverOptions, types.StringType, driverconst.CoreDNSImageTag, "coreDnsImageTag").(string),
+		OCICSIImage:          options.GetValueFromDriverOptions(driverOptions, types.StringType, driverconst.OCICSIImage, "ociCsiImage").(string),
+		CSIRegistry:          options.GetValueFromDriverOptions(driverOptions, types.StringType, driverconst.CSIRegistry, "csiRegistry").(string),
+		InstallCalico:        options.GetValueFromDriverOptions(driverOptions, types.BoolType, driverconst.InstallCalico, "installCalico").(bool),
+		InstallCCM:           options.GetValueFromDriverOptions(driverOptions, types.BoolType, driverconst.InstallCCM, "installCcm").(bool),
```
Comment (theabmitra) [inline ociocne/variables/variables.go:163]: uppercase installCCM ?

File: ociocne/variables/variables.go [hunk 2]
```diff
@@ -145,11 +152,16 @@ func NewFromOptions(ctx context.Context, driverOptions *types.DriverOptions) (*V
...
+		InstallCSI:           options.GetValueFromDriverOptions(driverOptions, types.BoolType, driverconst.InstallCSI, "installCsi").(bool),
```
Comment (theabmitra) [inline ociocne/variables/variables.go:165]: uppercase installCSI ?

Comment (anders-swanson) [reply ociocne/variables/variables.go:165]: this is how norman will format it

**Comment by anders-swanson** [comment]
@theabmitra could you re-review

**Review by theabmitra — APPROVED** [review]

**Pull request merged**